src/common/Webpack.ts

```diff
@@ -0,0 +1,62 @@
+import { FunctionImplementation, require } from "./index.js";
+import { IModImplementation } from "../api/ModImplementation";
+
+// const reservedData = {} as { [key: string]: any };
+
+export let targetMod: IModImplementation;
+
+const implementationStore = {
+    getModule: new FunctionImplementation({
+        depends: [],
+        supplies: "getModule",
+        // get data() {
+        //     return reservedData[this.supplies];
+        // },
+        // set data(v: any) {
+        //     reservedData[this.supplies] = v;
+        // },
+        data: {},
+        func(filter: (mod: any) => boolean) {
+            if (this.data.req == undefined) {
+                // @ts-expect-error Non-standard property
+                window.webpackChunkdiscord_app.push(
+                    [[Symbol()],
+                    {},
+                    (r: { c: any; }) =>
+                        this.data = this.data ?? {
+                            req: r,
+                        },
+                    ]);
+                // @ts-expect-error Non-standard property
+                window.webpackChunkdiscord_app.pop();
+            }
+            // @ts-expect-error too lazy
+            return Object.values(this.data.req).find(filter)?.exports;
+        },
+    }),
+    getByStrings: new FunctionImplementation({
+        depends: ["getModule"],
+        supplies: "getByStrings",
+        data: null,
+        func(...strings) {
+            const getModule = require(targetMod, "Webpack", "getModule");
+            if (getModule) {
```

Comment:
Swap the `if` to throw an error if there is no module (to avoid nesting)

Reply:
done in 05a7ed9